.github/workflows/ci.yml

```diff
@@ -11,13 +11,13 @@ on:
 
 jobs:
     validation:
-        name: "Static analysis / PHP 7.4"
+        name: "Static analysis / PHP 8.0"
         runs-on: ubuntu-18.04
         strategy:
             fail-fast: false
             matrix:
                 php-version: [
-                        '7.4',
+                        '8.0',
                 ]
 
         env:
```

Comment:
Please update the `node-version` version from `12` to `16` and the `Frontend install-dependencies` according to the [suite](https://github.com/spryker-shop/suite/blob/master/.github/workflows/ci.yml#L117)